src/HTTP/Middleware/Internal/NormalizeInfNanValues.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Is 'Normalize' really what this class is doing? I think the naming could work in other domains but since we're in the context of machine learning maybe we choose a different word. Normalize has a specific meaning in ML.

On the contrary, maybe InfNan is *too* specific. We might need to convert others in the future in which case the naming would not be valid. Naming is hard.

Would `convertRequestBodyConstants` work?